pom.xml

```diff
@@ -5,7 +5,8 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>tmc-auth</artifactId>
-  <version>LOCAL-SNAPSHOT</version>
+  <groupId>com.autonomic.tmc.auth</groupId>
+  <version>2.2.0</version>
```

Comment:
Do you have a specific reason for not letting `CI` set this?

Reply:
We were just testing. Changed back to LOCAL-SNAPSHOT